prow/config.yaml

```diff
@@ -5175,6 +5175,24 @@ presubmits:
         - "--upload=gs://kubernetes-jenkins/pr-logs"
 
   kubernetes-sigs/poseidon:
+  - name: ci-poseidon-e2e-gce
+    interval: 24h
```

Comment:
missing the `context` field for github

@cjwagner @stevekuznetsov do we really need separate "name" and "context" at this point ...?

Reply:
(leveraging them for branched jobs could be nice, but IIRC we've been enforcing that they are identical for a long time now...)

Reply:
I've been thinking that we should consider enforcing unique job names and support branch/RunIfChanged sharding by allowing jobs to specify the same context. I think that makes more sense intuitively and would could simplify the code in a number of places (having to search for the correct shard of a job based on the branch and changed files is cumbersome compared to looking up a job by its unique name).

There are some downsides to that though. I was hoping to get rid of the regexp triggers that jobs specify and instead require the job name to be used, but if we make different job shards have different names we would probably need to keep the regexp triggers around so that users don't have to worry about triggering the correct shard.

TLDR: As is, having both "name" and "context" isn't useful because they are always the same, but they do have semantic differences and I think it could be a good idea to keep them around because they may not always be the same if we change how sharding works.